.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: analysiscenter1/ds-py3:cpu
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build docs
```

Comment:
This is a duplication of `doc` workflow.

Reply:
Issue #490 says that release action should build docs, that's why it's here.

Since it may be a good idea too keep public docs up to date with latest public release, I suggest resolving this duplication by either total `doc` workflow removal, or by forcing it run only on releases (and remove building docs from this workflow).

Reply:
The doc should definitely be build and published during release, but `release` is not the only trigger